geth/api/backend.go

```diff
@@ -109,7 +111,14 @@ func (b *StatusBackend) startNode(config *params.NodeConfig) (err error) {
 			err = fmt.Errorf("node crashed on start: %v", err)
 		}
 	}()
-	err = b.statusNode.Start(config)
+
+	services := []gethnode.ServiceConstructor{
+		func(_ *gethnode.ServiceContext) (gethnode.Service, error) {
+			return mailservice.New(b.statusNode), nil
```

Comment:
isn't it better to add mailservice in the same place as shhext?

Reply:
Yeah, it should be more consistent.